README.md

```diff
@@ -172,3 +171,195 @@ Removed following and all unused discs from Azure Cloud.
 Removed following and all unused pips from Azure Cloud.
 ['test-axodawttrw-pip0']
 ```
+# How to run the cloudwash setup locally
+
+
+You will need a pod spec file, config map file (optional) and a secrets file. Create files locally named:
+
+* cloudwash.testpod.yaml:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cloudwash-test-pod
+spec:
+  containers:
+    - name: cloudwash
+      image: quay.io/redhatqe/cloudwash
+      command: ["/bin/sh"]
+      args: ["-c", "sleep 5000"]
+      volumeMounts:
+      - name: config-volume
+        mountPath: /opt/app-root/src/cloudwash/settings.yaml
+        subPath: settings.yaml
+      env:
+        - name: CLEANUP_AZURE__AUTH__CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              key: azure_client_id
+              name: cloudwash-secret
+        - name: CLEANUP_AZURE__AUTH__SECRET_ID
+          valueFrom:
+            secretKeyRef:
+              key: azure_client_secret
+              name: cloudwash-secret
+        - name: CLEANUP_AZURE__AUTH__TENANT_ID
+          valueFrom:
+            secretKeyRef:
+              key: azure_tenant_id
+              name: cloudwash-secret
+        - name: CLEANUP_AZURE__AUTH__SUBSCRIPTION_ID
+          valueFrom:
+            secretKeyRef:
+              key: subscription_id
+              name: cloudwash-secret
+  volumes:
+    - name: config-volume
+      configMap:
+        name: cloudwash-config
+  restartPolicy: Never
+```
+* cloudwash.configmap.yaml
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudwash-config
+data:
+  settings.yaml: |
+    AZURE:
+        AUTH:
+            CLIENT_ID: ""
+            SECRET_ID: ""
+            TENANT_ID: ""
+            SUBSCRIPTION_ID: ""
+            RESOURCE_GROUPS: []
+            REGIONS: []
+        CRITERIA:
+            VM:
+                DELETE_VM: ''
+                SLA_MINUTES: 120
+            DISC:
+                UNASSIGNED: True
+            NIC:
+                UNASSIGNED: True
+            IMAGE:
+                DELETE_IMAGE: ''
+                UNASSIGNED: True
+            PUBLIC_IP:
+                UNASSIGNED: True
+            RESOURCE_GROUP:
+                LOGIC: AND
+                DELETE_GROUP:
+                RESOURCES_SLA_MINUTES: 120
+        EXCEPTIONS:
+            VM:
+                VM_LIST: []
+                STOP_LIST: []
+            GROUP:
+                RG_LIST: []
+            IMAGES: []
+```
+* cloudwash.secrets.yaml
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloudwash-secret
+  namespace: default
+type: Opaque
+stringData:
+  azure_client_id: "XXXXXXXX"
+  azure_client_secret: "XXXXXXX"
+  azure_tenant_id: "XXXXXXXXX"
+  subscription_id: "XXXXXXXXX"
+```
+* You will also need to update the bonfire config.
```

Comment:
Hi @mshriver,
After you approved this MR I remembered I had made some changes in the bonfire local config too.
So, can you please review if it's required or not?

Reply:
I'd consider actually dropping this bonfire section, focusing on the minikube setup? IIRC there are issues with permissions in the ephemeral namespaces? I don't recall specific modifications made - because I think we ended up just using minikube to create the resources and pod.

Reply:
Right !!
I wasn't sure about it! 
anyways i just removed it 
@jyejare can you please merge it ?

Reply:
OOPs I did not notice the ping. Merging.